src/screens/select.tsx

```diff
@@ -108,42 +98,61 @@ const SelectScreen: React.FC<SelectScreenProps> = ({ navigation }) => {
         setStatusMessage(JSON.stringify(error));
       });
   };
-  const incAdult = () => {
-    if (adults < 6) {
-      setAdults(adults + 1);
-    }
-  };
-  const decAdult = () => {
-    if (adults > 0) {
-      setAdults(adults - 1);
-    }
-  };
-  const incChildren = () => {
-    if (children < 6) {
-      setChildren(children + 1);
-    }
-  };
-  const decChildren = () => {
-    if (children > 0) {
-      setChildren(children - 1);
+  const showOrHidePicker = (dateOrTime: boolean, show: boolean) => {
```

Comment:
I have no idea what this is doing and it looks like you might want to rename some of it/the parameters to make that a bit clearer